etc/config/circle.amazon.properties

```diff
@@ -0,0 +1,991 @@
+# Default settings for C++
+compilers=&circle:&circle128
```

Comment:
Thanks! I think I'll tweak these a bit rather than rely on the groups in the defaults; but I won't block merging on this.